config/config.inc.php

```diff
@@ -126,13 +125,20 @@
 define('__PS_BASE_URI__', $context->shop->getBaseURI());
 
 /* Include all defines related to base uri and theme name */
-require_once $currentDir.'/defines_uri.inc.php';
+require_once $currentDir . '/defines_uri.inc.php';
 
 global $_MODULES;
 $_MODULES = array();
 
-define('_PS_PRICE_DISPLAY_PRECISION_', Configuration::get('PS_PRICE_DISPLAY_PRECISION'));
-define('_PS_PRICE_COMPUTE_PRECISION_', _PS_PRICE_DISPLAY_PRECISION_);
+/**
+ * @deprecated since 1.7.7
+ */
+define('_PS_PRICE_DISPLAY_PRECISION_');
+
+/**
+ * @deprecated since 1.7.7
+ */
+define('_PS_PRICE_COMPUTE_PRECISION_', 6);
 
```

Comment:
This is the only part that changed in this file, the rest is php-cs-fixer doing his thing.